frontends/article/components/ArticleActions/article-action-icons.tsx

```diff
@@ -56,14 +63,14 @@ export const ArticleActionIcons: FC<IArticleActionsIconsProps> = ({
     dispatch(toggleCommentsSidebar());
   };
   const likeIcon = useMemo((): JSX.Element => {
-    if (isLikedOrDisliked === 0) {
```

Comment:
<img width="420" alt="Screenshot 2023-04-17 at 19 35 33" src="https://user-images.githubusercontent.com/55176773/232518909-b9f87caa-6eac-4d0e-a5cc-8806a0d61acc.png">
it should show like gif if article is not liked